### PR TITLE
docs: add support for transformerAttributifyJsx

### PR DESCRIPTION
**Pull request by valcosmos**

### CODE REVIEW

**Comment by zyyv** [comment]
I don't know how this helps to improve? I guess the usage instructions for `transformerAttributifyJsx` are enough in the [docs](https://unocss.dev/transformers/attributify-jsx).

File: examples/next/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "@types/react": "18.2.8",
     "@types/react-dom": "18.2.4",
     "@unocss/postcss": "latest",
+    "@unocss/preset-attributify": "latest",
```
Comment (zyyv) [inline examples/next/package.json:21]: You are use it import from `unocss`, why need install it again?

Comment (valcosmos) [reply examples/next/package.json:21]: It is quoted here, but it is not installed, so it does not need to be installed?

image

**Comment by valcosmos** [comment]
> I don't know how this helps to improve? I guess the usage instructions for `transformerAttributifyJsx` are enough in the [docs](https://unocss.dev/transformers/attributify-jsx).

u are right.

**Pull request closed without merging**